changelogs for core packages (#5403)
Azure/azure-sdk-for-js: sdk/core/core-http/Changelog.md

# Changelog

## 1.0.0-preview.4 - 2019-10-07

> - Stop exporting `@azure/core-tracing` from the library, instead have each library and eventually the customer import the tracing package explicitly.

Reviewer: ```suggestion
- No longer re-exports `@azure/core-tracing`. To enable tracing, call `setTracer` from `@azure/core-tracing` directly.
```

  ([PR #5389](https://github.com/Azure/azure-sdk-for-js/pull/5389))
- Report upload/download progress only after the first data chunk is received
  ([PR #5298](https://github.com/Azure/azure-sdk-for-js/pull/5298))

> - A new error type `ResponseBodyNotFoundError` is exported for cases when response body is empty.

Reviewer 2: ```suggestion
- Added new error type `ResponseBodyNotFoundError` for cases when the response body is unexpectedly empty.
```

  ([PR #5369](https://github.com/Azure/azure-sdk-for-js/pull/5369))

> - Temporary fix for a memoray leak issue resulting from creating new agents everytime Webresource is cloned.

Reviewer: ```suggestion
- Temporary fix for a memory leak issue resulting from creating new agents every time WebResource is cloned.
```

Reviewer 2: Chris, don't take away my Memo-Ray ®
